macOS/DuckDuckGo/Tab/View/BrowserTabViewController.swift

```diff
@@ -713,7 +715,10 @@ final class BrowserTabViewController: NSViewController {
         case .onboardingDeprecated:
             getView = { [weak self] in self?.transientTabContentViewController?.view }
         case .url, .subscription, .identityTheftRestoration, .onboarding, .releaseNotes, .history:
-            getView = { [weak self] in self?.webView }
+            getView = { [weak self, weak tabViewModel] in
+                guard let self, let tabViewModel else { return nil }
+                return webView(for: tabViewModel, tabContent: tabContent)
+            }
```

Comment:
this was sometimes racing resulting to a wrong `webView` (of a new tab) being added